src/go/k8s/api/vectorized/v1alpha1/console_types.go

```diff
@@ -82,6 +82,11 @@ type ConsoleSpec struct {
 
 	// The name of the ServiceAccount to be used by the Redpanda pods
 	ServiceAccount *string `json:"serviceAccount,omitempty"`
+
+	// +optional
+	// Console contains all configuration options for features that are generic,
+	// such as enabling API endpoints.
+	Console ConsoleConfigField `json:"console,omitempty"`
```

Comment:
consoleSpec.console does not sound right to me. 
Couldn't it be something like 
```
spec.dataplaneAPI = {
  enabled bool
  enabled_endpoints []string
}
```
?

Reply:
edit: given that console also has both enabled and enabled_procedures, we should ofc have hit as well

Reply:
based on https://github.com/redpanda-data/console/pull/955, the console: prefix should be removed right? so it's just 
```
API ConsoleAPI `json:"api" yaml:"api"`
```

without wrapping it into another Console

Reply:
I agree, but it will not be maintained in the future.

Reply:
Fixed

Reply:
@sago2k8 it looks like i said the wrong thing..i now just understood, that the console config layout actually looks like this. so we should actually have this field called console :/

I'm sorry for the noise. Could you bring it back to what you had initially?